test/helper.py

```diff
@@ -26,47 +26,52 @@ def login_as_device(self, devicename):
     def given_new_user(self, testClass, username):
         basename = self._make_long_name(testClass, username)
         testname = basename
-        for num in range(2,200): 
+        for num in range(2,200):
             try:
                 api = UserAPI(self.config.api_server, testname, self._make_password(testname)).register_as_new()
                 self._print_actual_name(testname)
                 return TestUser(testname, api)
             except Exception:
```

Comment:
I'm still not happy about catching Exception instead of something more general, but lets fix this later